knowledge-base/shivammathur/setup-php/action-security.yml

```diff
@@ -0,0 +1,83 @@
+name: 'setup-php'
+github-token:
+  environment-variable-name: COMPOSER_TOKEN
+  permissions:
+    contents: read
+    contents-reason: to access matching refs GitHub API, clone the repos, and get release assets.
+outbound-endpoints:
+  - fqdn: '*.launchpad.net'
+    port: 80
+    reason: to call launchpad API to get PPA fingerprint and get packages
+  - fqdn: '*.launchpad.net'
+    port: 443
+    reason: to call launchpad API to get PPA fingerprint and get packages
+  - fqdn: '*.ubuntu.com'
+    port: 80
+    reason: to download packages from ubuntu archives
+  - fqdn: '*.ubuntu.com'
+    port: 443
+    reason: to download packages from ubuntu archives
+  - fqdn: bazel.build
+    port: 443
+    reason: to download bazel GPG key
+  - fqdn: '*blackfire.io'
+    port: 80
+    reason: to set-up Blackfire
+  - fqdn: '*blackfire.io'
+    port: 443
+    reason: to set-up Blackfire
+  - fqdn: cdn.jsdelivr.net
+    port: 443
+    reason: to fetch scripts (fallback CDN)
+  - fqdn: deployer.org
+    port: 443
+    reason: to download Deployer phar
+  - fqdn: downloads.ioncube.com
+    port: 443
+    reason: to download ionCube extension
+  - fqdn: download.oracle.com
+    port: 443
+    reason: to download Oracle instantclient
+  - fqdn: getcomposer.org
+    port: 443
+    reason: to fetch Composer phars (fallback source)
+  - fqdn: git.remirepo.net
+    port: 443
+    reason: to download patches
+  - fqdn: '*.githubusercontent.com'
+    port: 443
+    reason: to fetch git objects, release information and code
+  - fqdn: '*github.com'
```

Comment:
Thanks @shivammathur for the updates! `*github.com` will be too broad. Can you please split it into two entries, one for `github.com` and one for `api.github.com`? Thanks!

Reply:
Done, split into `api.github.com`, `codeload.github.com` and `github.com`.

Reply:
Thanks!